R/localMaxima.R

```diff
@@ -0,0 +1,23 @@
+#' Local Maxima
```

Comment:
Some very minor thing that you can also ignore: I prefer to always use roxygen keywords, i.e. this would become `@title ...` and the description below `@description`. But as said, you can also simply ignore that suggestion.

Reply:
Ok, that's more descriptive. I will do that (but without opening a PR to review it).